src/router/accountRoutes.js

```diff
@@ -0,0 +1,26 @@
+import Pages from '../views/index';
```

Comment:
Why'd you separate this?

Reply:
@adetola-ralph it start to look long, so I trying to see if it might be better for us to eventually separate all of them. So I started with AccountRoutes.